src/App.tsx

```diff
@@ -16,35 +20,69 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.Default);
```

Comment:
The `SortType.Default` enum value should be in uppercase (`DEFAULT`) to follow the convention of enum values being constants.